gitlab_sync.go

```diff
@@ -51,9 +51,7 @@ func syncRemoteRef(log *logrus.Entry, org, repo, ref string, conf *config) error
 		}
 		// For the push, add option ci.skip for mender-qa
 		cmdArgs := []string{"push", "-f"}
-		if repo == "mender-qa" {
-			cmdArgs = append(cmdArgs, "-o", "ci.skip")
-		}
+		cmdArgs = append(cmdArgs, "-o", "ci.skip")
```

Comment:
@lluiscampos so in here we should set `ci.skip` for all the branches (GetHead()) excluding those (taken from the QA-336):
```
master
staging
production
[0-9]+\.[0-9]+\.x
pr_[0-9]+
```
right?